pom.xml

```diff
@@ -40,8 +40,6 @@
     </parent>
 
     <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
this is already present further down as well so was included twice

Reply:
This can potentially affect other sub projects, so i would like to check at a different timing. Could you revert the change?